src/algebra/field.lean

```diff
@@ -73,6 +73,9 @@ begin
   { exact ring.inverse_unit (units.mk0 x hx) }
 end
 
+lemma division_ring.val_inv_eq_inv_val (x : units K) : (x⁻¹).val = (x.val)⁻¹ :=
+units.coe_inv' x
+
```

Comment:
This lemma doesn't look like it should exist to me - the existing `coe_inv'` is already the preferred form.

Reply:
Ok, I'll remove it. It would be really nice to have documentation about normal forms for `units` in various context (including a bare monoid, a ring and a division ring).

Reply:
In the loosest possible sense of the word, this is "documented" by `units.simps.coe_inv`]

Reply:
Looking at this declaration I have no idea what it does. Hence it's hard to count this as "documentation".